ui/app/templates/permission_scheduled_changes.html

```diff
@@ -41,12 +41,12 @@
 
     <h3 class="panel-title">
       <div style="float: right">
-        <i ng-if=" scheduled_changes_rules_count && $first && (currentPage == 1)">Current</i>
-        <button class="btn btn-xs btn-primary" ng-click="signoff(sc)"
+        <i ng-show="!scheduled_changes_rules_count && !sc.complete" ng-if=" scheduled_changes_rules_count && $first && (currentPage == 1)">Current</i>
+        <button ng-show="!scheduled_changes_rules_count && !sc.complete" class="btn btn-xs btn-primary" ng-click="signoff(sc)"
```

Comment:
These shouldn't be conditional on sc.complete, just checking scheduled_changes_rules_count is enough. That being set or not is a proxy for "are we on the history page or not", and we _never_ want the buttons shown on the history page.